django/db/models/options.py

```diff
@@ -1,6 +1,6 @@
 import copy
 import inspect
-from bisect import bisect
+import bisect
```

Comment:
Please move this to the top of the list of imports to fix the error from `isort`.

Reply:
> Please move this to the top of the list of imports to fix the error from `isort`.

ok